examples/gocean/eg4/acc_transform.py

```diff
@@ -0,0 +1,81 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2018-2019, Science and Technology Facilities Council.
```

Comment:
Just 2019?

Reply:
Strictly speaking this is a copy of a pre-existing file so I've kept the date range.

Reply:
OK, that makes sense.